tasks/galen.js

```diff
@@ -153,35 +153,63 @@ module.exports = function (grunt) {
 
       var htmlReport = options.htmlReport === true ? '--htmlreport ' + (options.htmlReportDest || '') : '';
 
-      async.forEach(testFiles, function (filePath, cb) {
+      var maxLength = 0;
+      testFiles.forEach(function (filePath) {
+        maxLength = Math.max(maxLength, filePath.length);
+      });
 
+      var stack = testFiles.map(function (filePath) {
 
-        var command = ['galen test',
-          filePath,
-          htmlReport,
-          '-DwebsiteUrl="' + options.url + '"'
-        ].join(' ');
+        return function (cb) {
+          var command = ['galen test',
+            filePath,
+            htmlReport,
+            '-DwebsiteUrl="' + options.url + '"'
+          ].join(' ');
 
-        childprocess.exec(command, function (err, output, erroutput) {
-          if (err) {
-            return cb(err);
-          } else if (erroutput.replace(/\s/g, '')) {
-            
-            log('   • ' + filePath + ' failed'.red);
-            reports.push(erroutput);
+          // calculate number of spaces
+          var left = Math.abs(maxLength - filePath.length);
+          left += 4;
+          var spaces = [];
+          spaces[left] = ' ';
 
-            return cb();
-          }
+          grunt.log.write('    • ' + filePath + (spaces.join(' ')));
```

Comment:
- `grunt.log.write('    • ' + filePath + Array(Math.abs(maxLength - filePath.length) + 4).join(' ')  );`
- Remove unnecessary `var` declarations in the middle of the scope
- What is `4` standing for?